apps/prairielearn/elements/pl-drawing/elements.py

```diff
@@ -2407,14 +2405,14 @@ def get_attributes():
 
 class UnplaceableBaseElement(BaseElement):
     # Used only to get attributes
-    def generate(element):
-        raise Exception("Cannot create element!")
+    def generate(self):
+        raise RuntimeError("Cannot create element!")
 
     def is_gradable():
         return False
 
-    def grade(ref, st, tol, angtol):
-        raise Exception(
+    def grade(self, _st, _tol, _angtol):
+        raise NotImplementedError(
```

Comment:
Leaked change from another PR?

Reply:
Looks like it; should this be removed?

Reply:
Changed self to `element` in [b62be5d](https://github.com/PrairieLearn/PrairieLearn/pull/11104/commits/b62be5d6c6eb998f5a4a887abe0fddc2728f445e).